packages/cli/README.md

```diff
@@ -4,6 +4,14 @@ A modular tool to generate and manage Design Systems and other Accelerated Proje
 
 ## Usage
 
+## Available Commands:
+
+Run CLI without global install to generate a new design system folder :
+
+`npx radius-script init`
```

Comment:
we don't want to mention `radius-script` that is just for testing.

Reply:
Darn I thought that was the latest alpha version of the cli!